vb-fe/src/util/Endpoints.ts

```diff
@@ -10,3 +10,7 @@ export enum AuthEndpoints {
 export enum TagEndpoints {
   SEARCH = "/api/tag/search",
 }
+
+export enum RoomEndpoints {
+  GENERAL = "api/room/",
```

Comment:
I didn't want to call this CREATE, since it's just the general endpoint.  That being said, I didn't want to create its own `/api/room/create` endpoint either, since we should be able to GET POST and PUT the same endpoint.  GENERAL made the most sense in my mind, since its just the general endpoint

Reply:
yeah looks good.

I was thinking of making all of these endpoints into objects that has the structure
```
{
  path: string,
  method: POST | PUT | GET | DELETE
}
```

probably will do that in some refactor